fiftyone/utils/huggingface.py

```diff
@@ -651,6 +667,15 @@ def _parse_subset_kwargs(**kwargs):
     return subsets
 
 
+def _count_samples(sample_collection):
```

Comment:
This will be a bit more performant for grouped datasets:

```py
def _count_samples(sample_collection):
    if sample_collection.media_type == "group":
        sample_collection = sample_collection.select_group_slices(
            _allow_mixed=True
        )

    return sample_collection.count()
```

Reply:
Thanks for the pro-tip!